pom.xml

```diff
@@ -691,6 +691,8 @@
                     <artifactId>maven-javadoc-plugin</artifactId>
                     <version>${maven.javadoc.plugin.version}</version>
                     <configuration>
+                        <source>8</source>
```

Comment:
Could you please put a comment with explanation directly to this pom.xml. Otherwise I'm sure that somebody will change it in the future and will meet the same troubles.

Reply:
Added https://github.com/hazelcast/hazelcast/pull/24621/commits/98d512d23d85989c0efe8f4eba017a88df900b77